PoseEstimation/poseEstimation_main.py

```diff
@@ -0,0 +1,71 @@
+import os, requests
+import sys
+import numpy as np
+import h5py
+import typing
+from pathlib import Path
+import matplotlib.pyplot as plt
+from tifffile import imwrite, TiffWriter, TiffFile
+
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import poseEstimation_parameters as poseEst_params
+import poseEstimation_definitions as poseEst_defs
+
+import deeplabcut
+print("Imported DLC!", flush=True)
+
+# Import for PyInstaller
+from multiprocessing import freeze_support
+freeze_support()
+
+# Figure Style settings for notebook.
+import matplotlib as mpl
```

Comment:
Do we need matplotlib? We shouldn't use it, all the visualization has to happen in danse